test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -1133,6 +1149,106 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, IDPDiscoveryForm, B
         });
       });
     });
-
+    Expect.describe('Additional Auth Button', function () {
+      itp('does not show the divider and buttons if settings.customButtons is not set', function () {
```

Comment:
Can we add another test that checks the if customButtons is set we render the PIV button, the user name field and next button on the identity first page.

Reply:
Will add `expect(test.form.forgotPasswordLinkVisible()).toBe(false);` at line 1203